DEV-Simple/core/MediaManager.swift

```diff
@@ -15,6 +15,7 @@ import MediaPlayer
 class MediaManager: NSObject {
 
     weak var webView: WKWebView?
+    var devToURL: String
```

Comment:
Just out of curiosity for @benhalpern: Will the generalization efforts eventually encompass the mobile apps?

Reply:
Yeah we will in some capacity.

My vision is that we'll create our own native app which will act as a browser for all instances, as to not ask users to download many apps if they're part of the ecosystem. It would be a sort of "browser". When we cross that bridge I think we'd take all the code and approaches from this app and add some additional functionality.

We'll likely maintain it possible for folks to create their own apps, just like the DEV app, but we'll see what makes sense once we're facing down the problem.